src/test/java/core/basesyntax/strategy/OperationStrategyTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationStrategyTest {
+    private static final String BALANCE = "b";
+    private static final String PURCHASE = "p";
+    private static final String RETURN = "r";
+    private static final String SUPPLY = "s";
+    private OperationStrategy operationStrategy;
+    private OperationHandler balanceOperationHandler;
+    private OperationHandler purchaseOperationHandler;
+    private OperationHandler returnOperationHandler;
+    private OperationHandler supplyOperationHandler;
+    private Map<String,OperationHandler> operationHandlerMap;
+
+    @Before
+    public void setUp() throws Exception {
+        balanceOperationHandler = new BalanceOperationHandler();
+        purchaseOperationHandler = new PurchaseOperationHandler();
+        returnOperationHandler = new ReturnOperationHandler();
+        supplyOperationHandler = new SupplyOperationHandler();
+        operationHandlerMap = new HashMap<>();
+        operationHandlerMap.put(BALANCE,balanceOperationHandler);
+        operationHandlerMap.put(PURCHASE,purchaseOperationHandler);
+        operationHandlerMap.put(RETURN,returnOperationHandler);
+        operationHandlerMap.put(SUPPLY,supplyOperationHandler);
+        operationStrategy = new OperationStrategy(operationHandlerMap);
```

Comment:
It would be better to speed up all these variable declarations and put them in the map immediately.
```suggestion
    private OperationStrategy operationStrategy;
    private Map<String,OperationHandler> operationHandlerMap;

    @Before
    public void setUp() throws Exception {
        operationHandlerMap = new HashMap<>();
        operationHandlerMap.put(BALANCE, new BalanceOperationHandler());
        operationHandlerMap.put(PURCHASE, new PurchaseOperationHandler());
        operationHandlerMap.put(RETURN, new ReturnOperationHandler());
        operationHandlerMap.put(SUPPLY, new SupplyOperationHandler());
        operationStrategy = new OperationStrategy(operationHandlerMap);
```